components/automate-grpc/scripts/bootstrap_protoc.sh

```diff
@@ -14,11 +14,11 @@ set -e
 pushd components
 
 printf 'GEN: %s\n' components/automate-grpc/protoc-gen-a2-config/api/a2conf/*.proto
-protoc -I /src/components --go_out=logtostderr=true,paths=source_relative:/src/components \
+protoc -I /src/components --go_out=paths=source_relative:/src/components \
   automate-grpc/protoc-gen-a2-config/api/a2conf/*.proto
 
 printf 'GEN: %s\n' components/automate-grpc/protoc-gen-policy/iam/*.proto
-protoc -I /src/components --go_out=logtostderr=true,paths=source_relative:/src/components \
```

Comment:
the new golang proto compiler only logs to stderr. It does not recognize this option.